### ISSUE TITLE
Template adjustable corner on DC Blocker

### ISSUE THREAD

**Issue by baconpaul** [surge-synthesizer/surge-rack#614]
Lars notes: the S&H DC Blocker doesn't work that well

I bet if I make the blocking filter constant per-VCO templated we could crank that one up a bit. May help. Consider this before 1.0 but perhaps do it after.

**Comment by baconpaul** [comment]
<img width="1080" alt="Screen Shot 2022-11-11 at 9 39 14 AM" src="https://user-images.githubusercontent.com/13387561/201362726-540d7b89-60c5-4e21-8f34-18c2e59b19e0.png">

Hmm I'm not sure this is necessarily wrong - noise is full spectrum and this is basically shaped/filtered noise. Here's a comparison of it vs white noise in a spectrum for instance. 

@rubyglow curious what you were expecting here?

**Comment by baconpaul** [comment]
And pushing this to post 1.0 since i don't think the fix is a naive DC block for this particular module.

**Comment by rubyglow** [comment]
I see what you're saying Paul and technically you are correct. And probably most other noise modules don't block DC either. I guess I'm thinking ahead, or into a bigger musical context. Because how would I (users) actually use this module? I would most likely be sending it to a filter, and/or another effect, a VCA, an audio interface, etc.... And in **that** context, of how I would actually use it, DC-blocking would be quite useful, for all the usual reasons. But sure, I can use the NYSTHI DC blocker so it's not a biggie. It was more, since you already have that DC blocker in the context menu of the module, it would be nice to make it effective. Either that or maybe just remove it for the module.

**Comment by baconpaul** [comment]
yeah i agree. i want to be a bit careful though so i will look at differentially turning up the DC block here. But after the first version with the effective-for-everyone-else blocker is in the library.

As you can see, I really want to get to the library so i can get people off nightlies and start putting some features into the dev stream for early 23.

**Comment by baconpaul** [comment]
So I played around with this some

Basically a DC blocker that makes a difference on the output distorts the waveform way too much (and we already have a lo cut you could use).

I need to figure out a way other than the naive blocker. Surge VST has a very steep high pass in its signal path, and there is some attempt at DC compensation in the oscillator itself, but there's no 'easy' fix here. Pushing it back into not currently scheduled.
